packages/client/hmi-client/src/components/funman/tera-funman-boundary-chart.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+	<div ref="boundaryRef"></div>
+</template>
+
+<script setup lang="ts">
+import { ref, onMounted, watch } from 'vue';
+import {
+	getQueries,
+	processFunman,
+	RenderOptions,
+	renderFunmanBoundaryChart
+} from '@/services/models/funman-service';
+
+const props = defineProps<{
+	funModelId: string;
+	param1: string;
+	param2: string;
+	timestep: number;
+	options?: RenderOptions; // Should be optional
+}>();
+
+const boundaryRef = ref();
+const width = 60;
+const height = 60;
+const renderOptions = { width, height };
+
+onMounted(async () => {
+	const funModel = await getQueries(props.funModelId);
+	const processedData = processFunman(funModel);
```

Comment:
You are rendering this component multiple times with the same funman response, so instead of doing the same processing over and over I think we should just pass in the processed funman data structure instead of incurring repeated network and processing cost.

Reply:
Makes sense. I wasnt sure here whether or not having a massive ref would be better or worse than multiple calls

I will change this